src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -180,12 +180,14 @@ public Integer updateAppsUsageAssociationData(
      *
      * @param studyId       - the studyId
      * @param participantId - the participant
+     * @param deviceId - device id (unique to each combination of app-signing key, user, and device)
      * @return a list of neighbor entities and associations
      */
-    @GET( BASE + PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + APPS )
+    @GET( BASE + PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + DATASOURCE_ID_PATH + APPS )
```

Comment:
Rename `DATASOURCE_ID_PATH` to `DEVICE_ID_PATH`.

Reply:
I'm understanding `DEVICE_ID = some_function(DATASOURCE_ID)` .  Datasource is what is inputted in the API, and device_id is what we store.  This is correct and in line with the other endpoints.

Reply:
I think that should be a separate PR since DATASOURCE_ID_PATH is what's currently used in the code base, and renaming would require other unrelated changes in chronicle server.